README.md

```diff
@@ -21,24 +32,48 @@ Currently the following variables are supported:
 
 ### General
 
-* `wipefs_become` - Default: true. If this role needs administrator
+* `disk_wipe_become` - Default: true. If this role needs administrator
   privileges, then use the Ansible become functionality (based off sudo).
-* `wipefs_become_user` - Default: root. If the role uses the become
+
+* `disk_wipe_become_user` - Default: root. If the role uses the become
   functionality for privilege escalation, then this is the name of the target
   user to change to.
 
+* `disk_wipe_disks` - Default: [] empty list.
+  Extreme caution must be used to avoid running this role against the wrong disk(s)
+  Specifying the wrong disk or disks can lead to corruption and data loss.
+  Do not specify any disk that hosts your operating system or contains any data you may need.
+
 Dependencies
 ------------
 
 None
 
 Example Playbook
 ----------------
+This example will run against one disk dev/vdb.
+```yaml
+
+- hosts: disk_wipe-servers
+  roles:
+    - role: oasis_roles.disk_wipe
+      vars:
+      disk_wipe_disks: /dev/vdb
+
+```
 
+This example will run against a list of disks dev/vdb, /dev/vdc, dev/vdd
```

Comment:
In my opinion this is redundant.  I think that users of OASIS roles will know how lists work as a pre-requisite.

Reply:
updated